easybuild/easyconfigs/g/Gradle/Gradle-8.6.eb

```diff
@@ -0,0 +1,23 @@
+easyblock = 'PackedBinary'
+
+name = 'Gradle'
+version = '8.6'
+
+homepage = 'https://gradle.org'
+description = """Complete Gradle install.
+From mobile apps to microservices, from small startups to big enterprises,
+Gradle helps teams build, automate and deliver better software, faster.
+"""
+
+toolchain = SYSTEM
+
+source_urls = ['https://services.gradle.org/distributions']
+sources = ['gradle-%(version)s-all.zip']
+checksums = ['85719317abd2112f021d4f41f09ec370534ba288432065f4b477b6a3b652910d']
+
+sanity_check_paths = {
+    'files': ['bin/gradle'],
```

Comment:
Please include a sanity check command as well:

```
sanity_check_commands = ["gradle --help"]
```

Also, `Gradle` is a Java app, the `gradle` script uses the `java` command, so `Java` should be included as a runtime dependency (and the `versionsuffix` should be set accordingly)